src/tiled/commanddialog.h

```diff
@@ -78,8 +88,12 @@ private slots:
       */
     void removeSelectedCommands();
 
+    void currentChanged(const QModelIndex &current,
+                        const QModelIndex&) override;
+
 private:
     CommandDataModel *mModel;
+    CommandDialog *mDialog;
```

Comment:
Is there anyway of accessing some function of the parent QWidget instead of storing it in `mDialog` ?